include/SDL3_image/SDL_image.h

```diff
@@ -210,7 +210,7 @@ extern DECLSPEC void SDLCALL IMG_Quit(void);
  * by calling: SDL_SetSurfaceColorKey(image, SDL_RLEACCEL,
  * image->format->colorkey);
  *
- * If `freesrc` is SDL_TRUE, the RWops will be closed before returning,
+ * If `closeio` is SDL_TRUE, the RWops will be closed before returning,
  * whether this function succeeds or not. SDL_image reads everything it needs
  * from the RWops during this call in any case.
```

Comment:
There are multitudes of occurrences of `RWops` in there which I guess we should change to `IOStream`